### ISSUE TITLE
Namespaces.UnusedUses.UnusedUse doesn't work with some annotations

### ISSUE THREAD

**Issue by BafS** [slevomat/coding-standard#1465]
SlevomatCodingStandard.Namespaces.UnusedUses.UnusedUse doesn't work with some Nelmio annotation like `Model`. Example to reproduce:

```php
<?php
namespace App\Dto;

use OpenApi\Annotations as OA;
use App\Dto\Package; // <<< UnusedUse, should not
use Nelmio\ApiDocBundle\Annotation\Model; // <<< UnusedUse, should not

class MessageInfo {
    /**
     * @OA\Property(property="Packages", type="array",
     *     @OA\Items(type="object", ref=@Model(type=Package::class)),
     * ),
     * @Groups({"write-send"})
     */
    public int $packages;
}
```

**Comment by kukulich** [comment]
@BafS I'm not able to reproduce it. What version do you use? Can you prepare failing test?

**Comment by BafS** [comment]
I updated my issue @kukulich sorry, the `@Groups({"write-send"})` makes it fail. Apparently it's the `,` before `Groups` that makes it failing to be precise.

---

ruleset.xml:
```xml
<?xml version="1.0"?>
<ruleset name="AcmeProject">
    <config name="installed_paths" value="../../slevomat/coding-standard"/>
    <rule ref="SlevomatCodingStandard.Namespaces.UnusedUses">
        <properties>
            <property name="searchAnnotations" value="true"/>
        </properties>
    </rule>
</ruleset>
```

src/MessageInfo.php
```php
<?php
namespace App\Dto;

use OpenApi\Annotations as OA;
use App\Dto\Package; // <<< UnusedUse, should not
use Nelmio\ApiDocBundle\Annotation\Model; // <<< UnusedUse, should not
use Symfony\Component\Serializer\Annotation\Groups;

class MessageInfo {
    /**
     * @OA\Property(property="Packages", type="array",
     *     @OA\Items(type="object", ref=@Model(type=Package::class)),
     * ),
     * @Groups({"write-send"})
     */
    public int $packages;
}
```

Output:
```
./vendor/bin/phpcs --standard=ruleset.xml src/

FILE: /.../debug_slevomat/src/MessageInfo.php
-------------------------------------------------------------------------------------
FOUND 2 ERRORS AFFECTING 2 LINES
-------------------------------------------------------------------------------------
 5 | ERROR | [x] Type App\Dto\Package is not used in this file.
 6 | ERROR | [x] Type Nelmio\ApiDocBundle\Annotation\Model is not used in this file.
-------------------------------------------------------------------------------------
PHPCBF CAN FIX THE 2 MARKED SNIFF VIOLATIONS AUTOMATICALLY
-------------------------------------------------------------------------------------
```

**Comment by kukulich** [comment]
Improved in https://github.com/slevomat/coding-standard/commit/c1ba31f8efdbd98c6bf44ec211b09a45bc531480

**Comment by BafS** [comment]
Thanks a lot @kukulich!
